templates/idempotent-wrapper.sql

```diff
@@ -1,8 +1,17 @@
 
 -- {{filename}}
 DO $outer_idempotent_block$
+DECLARE
+    _old_search_path text;
 BEGIN
+    EXECUTE 'SHOW search_path' INTO STRICT _old_search_path;
+    SET search_path TO pg_catalog;
```

Comment:
we should just be able to use SET local here and avoid having to restore

Reply:
as above